WordPress/src/main/java/org/wordpress/android/ui/comments/unified/UnifiedCommentListFragment.kt

```diff
@@ -167,7 +173,27 @@ class UnifiedCommentListFragment : Fragment(R.layout.unified_comment_list_fragme
     val commentDetails = registerForActivityResult(CommentDetailsActivityContract()) { response:
     CommentDetailsActivityResponse? ->
         if (response != null) {
-            viewModel.performSingleCommentModeration(response.commentId, response.commentStatus)
+            viewLifecycleOwner.lifecycleScope.launchWhenStarted {
+                if (sharedPreferences?.getString(
+                        COMMENT_STATE_KEY,
+                        COMMENT_STATE_INITIAL
+                    ) == COMMENT_STATE_SUCCESSFULLY_EDITED
+                ) {
+                    viewModel.localCommentCacheUpdateHandler.requestCommentsUpdate()
```

Comment:
I don't think this is the best way to do it, to be honest. No one needs to know that VM is using `localCommentCacheUpdateHandler`. If you want the logic to be related to VM, I would expose a public method from the VM, that will do the update. This way we can at least test it.

If you think that the logic is not related to VM, I would have just used a different instance `localCommentCacheUpdateHandler` not exposed from VM.

Reply:
I'll check that, thanks 😄